### ISSUE TITLE
How to run live query and cloud code?

### ISSUE THREAD

**Issue by rayyoungnext** [bitnami/bitnami-docker-parse#41]
First of all, thanks very much for the docker image. It's really very each to get the parse server up and running. However, i couldn't find any information regarding how to run the live query and cloud code with this docker image. It will be great if you can provide some tips on how to run live query and cloud code with this docker image. Thanks.

**Comment by paulfreeman** [comment]
I also have this question.  There is an example of cloud-code available from https://github.com/yongjhih/docker-parse-server maybe that could help

**Comment by chipironcin** [comment]
Hello @rayyoungnext and @paulfreeman 
It seems Parse LiveQuery and Parse Cloud Code are two features that would make sense to support and have documented.

We will work towards it and update this issue with news.
Thanks for the request!

**Comment by rayyoungnext** [comment]
Thanks for the update. Looking forward to seeing the live query and cloud code coming soon.

**Comment by mnlbox** [comment]
@chipironcin any news?

**Comment by andresbono** [comment]
Hi @rayyoungnext, @paulfreeman, @mnlbox:

We have tried a simple example to test the _CloudCode_ functionality in our container. These are the steps that we have followed:

1. In the parse container, execute:
  `mkdir -p /opt/bitnami/parse/cloud`.

2. Create a simple CloudCode script (`/opt/bitnami/parse/cloud/main.js`) with the following content:
```
Parse.Cloud.define("sayHi", function(request, response) {
    response.success("Hi there!");
});
```

3. Reference the script in the parse config file (`/opt/bitnami/parse/config.json`).
```diff
 {
   ...
   "port": "1337",
   "host": "0.0.0.0",
+  "cloud": "./cloud/main.js",
   "serverURL": "http://127.0.0.1:1337/parse",
   ...
 }
```

4. Restart Parse:
```
nami restart parse
```

5. Access Parse Dashboard API Console http://localhost/apps/MyDashboard/api_console and do the query:

![image](https://user-images.githubusercontent.com/1057973/43516547-273eb476-9586-11e8-96f2-41fbff105425.png)


We will include this information in the README, but posting here in the meantime.
-Andrés B.

**Comment by mnlbox** [comment]
@andresbono the missing part for me is number one:
I create cloud directory in my host not in container.

> In the parse container, execute:
> mkdir -p /opt/bitnami/parse/cloud.

I think it's better to update sample `docker-compose.yml` exist in ReadMe.md file to support cloud code too.

**Comment by andresbono** [comment]
Thanks for your feedback @mnlbox, it sounds like a good idea. We will update this issue once we implement it in our container.

**Comment by mnlbox** [comment]
@andresbono any news about this?

**Comment by andresbono** [comment]
Hi @mnlbox, we didn't forget about it, but this is still not addressed. We will update this issue as soon as we implement the change.

Thanks for your patience.

**Comment by mnlbox** [comment]
@andresbono still waiting...

**Comment by andresbono** [comment]
Hi @mnlbox, we have scheduled the task to implement your feedback for the next week.

Hope we have more news soon!

**Comment by juan131** [comment]
Hi @mnlbox 

We just released a new version of the image (**3.1.2-r2**) including a new env. variable (**PARSE_ENABLE_CLOUD_CODE**) which allows you to easily configure CloudCode.

We also include a section describing the whole process to include your own Cloud functions and run them on this Docker image. Find them in the link below:

https://github.com/bitnami/bitnami-docker-parse#how-to-deploy-your-cloud-functions-with-parse-cloud-code

I hope it helps you!

**Comment by mnlbox** [comment]
Thanks @juan131 I'm going to try this. :wink:
